articles/introduction-to-deep-learning/index.md

```diff
@@ -0,0 +1,123 @@
+
+Deep Learning algorithms strive to reason the same as humans by studying data with a particular [logical framework](https://medium.com/tebs-lab/introduction-to-deep-learning-a46e92cb0022).
+
+### Machine Learning vs. Deep Learning
+Deep Learning and machine learning are both types of AI that classify data and train models.
+
+1. Data presentation in the system. Machine Learning algorithms need structured data to solve data problems. Deep Learning relies on layers of Neural Networks.
+Example: you have several animal images, such as birds and pets. A person would label each image to help an ML algorithm differentiate the images. Deep Learning is a bit complex. This is because using the same dataset. Deep Learning algorithms toil to evaluate variations between categories and classify them.
+
+2. A Machine Learning algorithm can take a minute or up to a few hours to finish execution. Deep Learning algorithms can go up to weeks to halt the execution. This is due to the complexity of Neural Networks. Neural networks need a large number of weights and more parameters.
+
+3. The performance of ML algorithms is better when the dataset is small or medium. Deep Learning algorithms fail to perform well on such datasets. DL algorithms instead perform exceptionally well for bigger datasets.
+
+4.	Machine Learning algorithms can perform on inexpensive. Deep Learning is a potent tool. It performs well on a powerful machine equipped with multiple GPUs, thus [providing high performance](https://hackr.io/blog/machine-learning-vs-deep-learning).
+
+5. Machine Learning algorithms study and understand labeled data in a dataset. Deep Learning algorithms require no human intervention. This is because layers in Neural Networks place data in different concepts. Neural Networks also enable DL algorithms to learn independently.
+
+6. Deep Learning is an essential part of Machine Learning. It provides solutions to many problems in image recognition, speech recognition, and NLP. Deep Learning performs well with images, text, sound recognition as compared to Machine Learning.
+
+### How Deep Learning Works
+The functioning of the brain influences deep Learning algorithms. Neural Networks are made up of layers and nodes, as well as the brain comprises of neurons.
+
+Neural Networks consist of:
+- Input layer
+- Hidden layers
+- Output layer
+
+The input layer receives input data. Hidden layers perform mathematical computations on the information. The output layer gives the result. The problem is solved based on computation from the distributive [weights of all layers](https://www.modev.com/blog/how-deep-learning-works).
+
+A good example to use is the recognition of a house image. The first layer might focus on borderlines. Another layer focus on the object colors, and another the measurements of the image.
```

Comment:
You need to mention the use of CNNs here. If not, the reader might get confused with ANNs. Such varied feature extraction takes place in CNNs.  Also if possible, include an image of a neural network showing the feature extraction process.